Dockerfile

```diff
@@ -1,8 +1,20 @@
-FROM fabric8/java-jboss-openjdk8-jdk:1.5.1
+FROM --platform=$BUILDPLATFORM eclipse-temurin:18-jdk-focal as builder
```

Comment:
Why `-platform=$BUILDPLATFORM` is needed?

Reply:
It saves time not pulling the image for each one of the architectures. We pull just for the architecture where we are building.